OneCommunityGlobal/HighestGoodNetworkApp
# Yuri - Design navigation bar

# Description
This PR is associated with the design of the navigation bar for Phase II of HGN Software. Two elements require testing: (i) the standard Header, which is currently utilized and (ii) the prototype (provided below) included in HGN Phase II for Materials, Equipment, and Tools.

The current Header, featuring the new navbar connecting BMDashboard and Project from HGN Software Phase 2 implementation, is displayed in the attached image.

![Screenshot 2023-10-14 115246](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/121908988/c423db5d-8f8f-4f4c-b3e5-4a9be247b45c)

The prototype below will be integrated into the existing Header in the future, with the links being managed through states. To test this, you should comment out the Header and uncomment the BMHeader in the “src/routes.js” file.



![Screenshot 2023-10-06 145548](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/121908988/5e9530b9-b281-4137-ae6c-9b7d493406c6)

![Screenshot 2023-10-06 145747](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/121908988/c7412b62-f73f-4872-88c7-e4c257cb2967)


## How to test:
1.0 Current Header component
1.1 Check out the current branch.
1.2 Run "git pull".
1.3 Run "npm install" and "npm run start:local" to run this PR locally.
1.4 Go to http://localhost:3000/
1.5 Verify the normal functioning of the following elements: owner message, timer, login/logout, and ensure that the BMDashboard and all new Project links are operational.
1.6 Check if there are any errors.
_____________________________________

2.0 To test the BMHeader, please comment out the Header and uncomment the BMHeader in the “src/routes.js” file.
2.1 Compare the design to the prototype.
2.2 Test for responsiveness in different screen sizes.
2.3 Verify that the reports, userprofile and login/logout functionality are working normally.
2.4 Check the other navbar links; they should redirect you to blank pages that will be developed further as part of the Phase 2 project.
2.5 Check if there are any errors.

## After the PR was moved to the final review, Jae told me to transfer all the logic from the Header component, which contains the navbar phase 2 features, to the BMHeader component. This was done to hide the links to the P2 workplace until its completion. To work during phase 2, developers will need to manually comment out the Header component and uncomment the BMHeader component in routes.js in order to view the navbar links for phase 2.

## After:

![Screenshot 2023-10-13 124527](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/121908988/af549909-cc27-4753-8f18-f5a160ff3081)


![Screenshot 2023-10-06 145022](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/121908988/c2a43785-45ae-4dab-bb2f-306b2650dd77)
